composer.json

```
@@ -13,7 +13,7 @@
         "sabre/uri"    : "^2.0"
     },
     "require-dev" : {
-        "phpunit/phpunit" : "^6.0 || ^7.0"
+        "phpunit/phpunit" : "^6.0 || ^7.0 || ^8.0"
```

for 7.1+ we no longer need phpunit 6.x

```suggestion
        "phpunit/phpunit" : "^7.0 || ^8.0"
```

---

Did you intentionally remove the `^` on version 7?